src/views/shared/FooterSignout.js

```diff
@@ -46,8 +46,8 @@ export default View.extend({
   },
   getTemplateData: function() {
     return {
-      linkClassName: _.isUndefined(this.options.linkClassName) ? 'goto' : this.options.linkClassName,
```

Comment:
why remove `goto`

Reply:
`.goto` will change float to right.
Right float seems good for 'Sign out', but for 'Back to sign in' left float seems more appropriate (because of "Back" word).
Need thought from UX.